changelog/security/2022-5-24-remove-autologin-for-packet-oem.md

```diff
@@ -0,0 +1 @@
+* Remove `flatcar.autologin` as a default kernel argument for best practices on packet-oem overlay.
```

Comment:
```suggestion
- Equinix Metal: Remove `flatcar.autologin` as a default kernel argument for making it harder to get system access through the serial console. Users that want to use the OOB console have to either set up a user password (and make sure that SSH password login is disabled) or set the kernel argument when provisioning through the Ignition v3 kargs setting (requires Flatcar major version >=3185) ([coreos-overlay#1914](https://github.com/flatcar-linux/coreos-overlay/pull/1914))
```

Reply:
Also, I think the `changelog/changes/` folder fits this better as no security issue was addressed but an OS change is done that requires users to adapt to the change